Practice2/README.md

```diff
@@ -22,6 +22,7 @@
 *Step two*: How to run the project
 * Turn on Visual Studio Code. Open the folder you just cloned to your computer.
 * Select Terminal on the toolbar or press Ctrl + Shift + `
+* Run command "cd Internship"
```

Comment:
Should add one more step: git checkout practice-two

Reply:
Fixed